src/Symfony/Component/Console/Command/TraceableCommand.php

```diff
@@ -86,9 +88,14 @@ public function __call(string $name, array $arguments): mixed
         return $this->command->{$name}(...$arguments);
     }
 
-    public function getSubscribedSignals(): array
+    public function getSubscribedSignals(/* InputInterface $input, OutputInterface $output */): array
```

Comment:
A final class should add the arguments directly instead of such forward compatibility layer, given that it does not have child classes that prevent adding them.

Reply:
This is no longer an issue since I've removed that commit.